core/src/main/java/org/apache/accumulo/core/client/mapred/AbstractInputFormat.java

```diff
@@ -484,7 +509,7 @@ public void initialize(InputSplit inSplit, JobConf job) throws IOException {
       if (null == authorizations) {
         authorizations = getScanAuthorizations(job);
       }
-
+      String classLoaderContext = getContext(job);
```

Comment:
Well. yes? If the variable is treated as immutable, we should always mark it as such to avoid potential errors.